types/d3-color/index.d.ts

```diff
@@ -91,6 +115,15 @@ export interface LabColorFactory extends Function {
     readonly prototype: LabColor;
 }
 
+/**
+ * Constructs a new Lab color with the specified l value and a = b = 0.
+ */
+// export interface GrayColorFactory extends Function {
+//     (l: number, opacity?: number): LabColor;
+// }
```

Comment:
Avoid commented code.

Reply:
Actually left on purpose. I used a type as I got an error using the interface. But do we want a type, or add an exception rule to allow for the interface?